src/account_multisig.ts

```diff
@@ -245,7 +293,21 @@ export class Account2FA extends AccountMultisig {
                     perm.method_names.includes('add_request_and_confirm');
             });
         const confirmOnlyKey = PublicKey.from((await this.postSignedJson('/2fa/getAccessKey', { accountId })).publicKey);
+        await this.deleteAllRequests().catch(e => e);
+        const currentAccountState: { key: Buffer, value: Buffer }[] = await this.viewState('').catch(error => {
+            const cause = error.cause && error.cause.name;
+            if (cause == 'NO_CONTRACT_CODE') {
+                return [];
+            }
+            throw cause == 'TOO_LARGE_CONTRACT_STATE' ? this.contractHasExistingStateError : error;
+        });
+        const currentAccountStateKeys = currentAccountState.map(({ key }) => key.toString('base64'))
+        const cleanupActions = currentAccountState.length ? [
+            deployContract(cleanupContractBytes),
+            functionCall('clean', { keys: currentAccountStateKeys }, MULTISIG_GAS, new BN('0'))
```

Comment:
It's about 101 Tgas based on my testing with 100 Tgas being attached to the `functionCall` . I'm not sure if this is significant given the 300 Tgas limit and whether it would make sense to attach less gas to the `clean` function call. [here](https://explorer.testnet.near.org/transactions/9wzLLWdQHozXL6WPe66A3gPW6R1gz89ej5cQeb1SeMPV)'s a basic disable transaction with no requests on the multisig.